doc/en/user/source/production/container.rst

```diff
@@ -15,13 +15,41 @@ Set the following performance settings in the Java virtual machine (JVM) for you
+     - Defines an upper limit on how much heap memory Java will request from the operating system  (use more if you have excess memory). By default, the JVM will use 1/4 of available system memory. The setting ``-Xms756m`` allocates 756MB of memory to GeoServer.
+   * - ``-XX:MaxPermSize=512m``
+     - This setting is **no longer needed for Java 8** to to a change in how applications are loaded. Previously we asked administrators to rase this value as GeoServer is a relatively large application.
    * - ``-XX:SoftRefLRUPolicyMSPerMB=36000``
-     - Increases the lifetime of "soft references" in GeoServer.  GeoServer uses soft references to cache datastore references and other similar requests.  Making them live longer will increase the effectiveness of the cache.
+     - Increases the lifetime of "soft references" in GeoServer.  GeoServer uses soft references to cache datastore, spatial reference systems, and other data structures. By increasing this value to ``36000`` (which is 36 seconds) these values will stay in memory longer increasing the effectiveness of the cache.
    * - ``-XX:+UseParallelGC``
-     - Enables the throughput garbage collector.
+     - The default garbage collector, **pauses the application while using several threads to recover memory**. Recommended if your GeoServer will be under light load and can tolerate pauses to clean up memory.
+   * - ``--XX:+UseParNewGC``
+     - Enables use of the concurrent mark sweep (CMS) garbage collector **uses multiple threads to recover memory while the application is running**. Recommended for GeoServer under continuous use, with heap sizes of less than 6GB.
+   * - ``–XX:+UseG1GC``
+     - Enables use of the `Garbage First Garbage Collector (G1) <http://www.oracle.com/technetwork/java/javase/tech/g1-intro-jsp-135488.html>`_ using **background threads to scan memory while the application is running** prior to cleanup. Recommended for GeoServer under continuous load and heap sizes of 6GB or more. Additionally you may experiment with ``-XX:+UseStringDeduplicationJVM`` to ask G1 to better manage common text strings in memory.
 
-For more information about JVM configuration, see the article `Performance tuning garbage collection in Java <http://www.petefreitag.com/articles/gctuning/>`_.
+For more information about JVM configuration, see the article `Performance tuning garbage collection in Java <http://www.petefreitag.com/articles/gctuning/>`_ and `The 4 Java Garbage Collectors <http://blog.takipi.com/garbage-collectors-serial-vs-parallel-vs-cms-vs-the-g1-and-whats-new-in-java-8/>`_.
+
+.. note:: 
+   
+   If you're serving just vector data, you'll be streaming, so having more memory won't increase performance.  If you're serving coverages, however, image processing will use a disk cache and benifit from more memory. It is also a good idea to configure the JAI tile cache size (see the Server Config page in the :ref:`web_admin` section) so that it uses 75% of the heap (0.75).
```

Comment:
I agre about the disk cache, I copied that text out ... let me check what it was before.

Check it out (http://docs.geoserver.org/latest/en/user/production/container.html#optimize-your-jvm) it says disk cache currently.